tensorflow/lite/micro/tools/ci_build/test_esp32.sh

```diff
@@ -26,34 +26,34 @@ pwd
 source tensorflow/lite/micro/tools/ci_build/helper_functions.sh
 
 TARGET=esp
+TARGET_ARCH=xtensa-esp32
 
-# setup esp-idf and toolchains
-echo "Checking out esp-idf..."
-readable_run git clone --recursive --single-branch --branch release/v4.2 https://github.com/espressif/esp-idf.git
-export IDF_PATH="${ROOT_DIR}"/esp-idf
-cd $IDF_PATH
-readable_run ./install.sh
-readable_run . ./export.sh
-cd "${ROOT_DIR}"
```

Comment:
While this is removed out there is another one above on line 23 that puts us into the ROOT_DIR (/opt/tflite-micro) before the commands are run.